### PR TITLE
Add utility things

### PR DESCRIPTION
**Pull request by II-KA**
Added utility stuff so I can also get to writing tests: 

- babel preset to compile ES2015 to ES5
- jest-extended to test setup
- automatic code linting and formatting

(few things are cherry-picked from first-tests branch)

### CODE REVIEW

**Comment by II-KA** [comment]
About prettier:

"Prettier is an opinionated code formatter. It enforces a consistent style by parsing your code and re-printing it with its own rules that take the maximum line length into account, wrapping code when necessary."

I'd recommend installing the "Prettier - Code formatter" extension in vscode and setting it to format on save. Instructions are available in the extension page.

Imo will make our code more readable and eliminate need to comment on coding style in merge requests.

+ format on save is fun to use 🙂

File: package.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "type": "module",
   "scripts": {
     "test": "jest --coverage"
```
Comment (Liljenmaa) [inline package.json]: Add lint check here has well

**Review by Liljenmaa — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -3,15 +3,18 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "type": "module",
   "scripts": {
-    "test": "jest --coverage"
+    "test": "jest --coverage",
+    "lint": "eslint test/** --max-warnings=0"
```
Comment (II-KA) [inline package.json:8]: Configured the lint command to only check test folder and moved the dummy test there

Comment (Liljenmaa) [reply package.json:8]: Remember to also include this in .github/workflows/node.js.yml so that it runs also in the pipeline.

**Review by Liljenmaa — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (II-KA) [reply package.json:8]: Added, good catch

**Pull request merged**